faqs/galaxy/tools_missing.md

```diff
@@ -3,26 +3,46 @@ title: If a Tool is Missing
 area: tools
 box_type: tip
 layout: faq
-contributors: [shiltemann]
+contributors: [shiltemann, garimavs]
 ---
 
+To use the tools installed and available on the Galaxy server:
+
+1. At the top of the left tool panel, type in a tool name or datatype into the tool search box.
+2. Shorter keywords find more choices.
+3. Tools can also be directly browsed by category in the tool panel.
+
+If you can't find the tool you want:
+
+1. Try searching the [Tool Shed](http://toolshed.g2.bx.psu.edu/).
```

Comment:
Thanks for your contribution @garimavs!

I don't think the second step here should be searching the Tool shed and talking about wrapping of tools. This FAQ was meant for GTN tutorials, so we know they should be installed on the server (if they made sure to use a compatible Galaxy), but the user is just not able to find it.

But maybe this could go into a separate FAQ, something like "adding new tools" orso? What do you think?

Reply:
Hello @shiltemann, I once considered that adding this content might not bear much co-relation, but then I felt that rather than only referring to GTN tutorials, we modify it such that every case of missing/can't find tools comes under this FAQ.

Though I will consider your insight and open a separate FAQ regarding "adding new tools."

Reply:
@garimavs I understand your reasoning, but this FAQ is added to every individual tutorial FAQ pages, where people who are following a tutorial will go when they get stuck, and this is one of the most common questions, so it would be best if only the most relevant answers for that audience are here. 

Having a (different) FAQ as a landing page for all "missing tool" questions, like you suggest, could be a good idea, then that can link out to this and other FAQs, e.g. "Can't find a tool for a GTN tutorial?" (link to this one). as well as some of these other cases you describe, such as requesting installation of toolshed tools to a server, or adding new tools to the toolshed.

Reply:
> @garimavs I understand your reasoning, but this FAQ is added to every individual tutorial FAQ pages, where people who are following a tutorial will go when they get stuck, and this is one of the most common questions, so it would be best if only the most relevant answers for that audience are here.
> 
> Having a (different) FAQ as a landing page for all "missing tool" questions, like you suggest, could be a good idea, then that can link out to this and other FAQs, e.g. "Can't find a tool for a GTN tutorial?" (link to this one). as well as some of these other cases you describe, such as requesting installation of toolshed tools to a server, or adding new tools to the toolshed.

Yes, indeed, we can work on something along the same lines.